package.sh

```diff
@@ -1,30 +1,28 @@
 #!/bin/bash -e
 
 rm -rf node_modules
+
 if [ -z "${ADDON_ARCH}" ]; then
   TARFILE_SUFFIX=
 else
   NODE_VERSION="$(node --version)"
   TARFILE_SUFFIX="-${ADDON_ARCH}-${NODE_VERSION/\.*/}"
 fi
-# For openwrt-linux-arm and linux-arm we need to cross compile.
-if [[ "${ADDON_ARCH}" =~ "linux-arm" ]]; then
-  # We assume that CC and CXX are pointing to the cross compilers
-  npm install --ignore-scripts --production
-  npm rebuild --arch=armv6l --target_arch=arm
-else
-  npm install --production
-fi
 
-rm -f SHA256SUMS
-sha256sum package.json manifest.json *.js LICENSE README.md > SHA256SUMS
-rm -rf node_modules/.bin
-find node_modules -type f -exec sha256sum {} \; >> SHA256SUMS
-TARFILE="$(npm pack)"
+npm install --production
+
+shasum --algorithm 256 manifest.json package.json *.js LICENSE README.md > SHA256SUMS
+
+find node_modules \( -type f -o -type l \) -exec shasum --algorithm 256 {} \; >> SHA256SUMS
```

Comment:
Did you find symlinks outside the `.bin` directory?

Reply:
No, but this is a much less fragile solution.

Reply:
I see.
I'll use this for all future package scripts.